compiler-rt/lib/sanitizer_common/sanitizer_libc.cpp

```diff
@@ -279,6 +279,25 @@ uptr internal_wcsnlen(const wchar_t *s, uptr maxlen) {
   return i;
 }
 
+wchar_t* internal_wcscpy(wchar_t* dst, const wchar_t* src) {
```

Comment:
I would rather put this next to char* version and make sure implementation looks consistent.

Reply:
I will note that `wcslen` is also quite far from `strlen`; we may eventually want to move these together.

Reply:
If you move this into a separate patch, feel free to reorder those as well.